src/seeds.accounts.cpp

```diff
@@ -259,25 +263,12 @@ void accounts::refreward(name account, name new_status) {
     return;
   }
 
-  // Add community building point +1
+  // Add community building points
 
   name cbp_param = is_citizen ? cbp_reward_citizen : cbp_reward_resident;
+  int community_building_points = int(config_get(cbp_param));
 
-  auto community_building_points = config_get(cbp_param);
-
-  auto citr = cbs.find(referrer.value);
-  if (citr != cbs.end()) {
-    cbs.modify(citr, _self, [&](auto& item) {
-      item.community_building_score += community_building_points;
-    });
-  } else {
-    cbs.emplace(_self, [&](auto& item) {
-      item.account = referrer;
-      item.community_building_score = community_building_points;
-      item.rank = 0;
-    });
-    size_change("cbs.sz"_n, 1);
-  }
+  add_cbs(referrer, community_building_points);
```

Comment:
The referrer should not get additional CBP - they are already included with the vouch rewards?

Reply:
Got it.